docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
 
   gitlab:
     build: gitlab/
+    volumes-from: gitlab-data
```

Comment:
im not terribly concerned about this now, also I can see use cases for importing data outside of building it (such as modifying preexisting containers)